src/app/models/body-type.ts

```diff
@@ -2,3 +2,16 @@ export interface BodyType {
   code: string;
   description: string;
 }
+
+export interface IBody {
```

Comment:
Just commenting on this naming convention as we are trying to make it similar to other models.